client/verta/verta/_repository/commit.py

```diff
@@ -632,13 +632,45 @@ def merge(self, other, message=None):
         elif self._repo.id != other._repo.id:
             raise ValueError("Commit and `other` must belong to the same Repository")
 
-        if message is None:
-            message = "Merge {} into {}".format(
-                other.branch_name or other.id[:7],
-                self.branch_name or self.id[:7],
+        msg = _VersioningService.MergeRepositoryCommitsRequest()
+        if self.branch_name is not None:
```

Comment:
I'm assuming you reviewed a vs b because I don't recall right now =)

Reply:
Yep, I've triple-checked this 😵